BUILD.bazel

```diff
@@ -191,7 +191,54 @@ config_setting(
 )
 
 container_bundle(
-    name = "build-other-images",
+    name = "build-other-images_s390x",
+    images = {
```

Comment:
I think  `select` should work here and it would look more like Bazel. Also it will be clear that if one modifies one arch it needs to modify other...

Reply:
@jschintag we should remove the usage of default, ARM might be messed up in this way as well.

Reply:
Select was my first choice here as well, unfortunately bazel did not accept it in this field...

Reply:
I'll have a look why is this the case later but it should work afaik